### PR TITLE
Imp wave2 980 drawer fixes

### PR DESCRIPTION
**Pull request by TDRoden**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss [hunk 1]
```diff
@@ -554,6 +554,16 @@ WATCH THE VIDEO BUTTON ALIGNMENT
       }
     }
```
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss]: This file not needed, right?

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_drawer/ucgy_gbl_drawer.info [hunk 2]
```diff
@@ -18,6 +18,8 @@ dependencies[] = mini_panel_reference
 dependencies[] = options
 dependencies[] = text
 dependencies[] = views
+stylesheets[all][] = css/ucgy_gbl_drawer.css
+scripts[] = js/ucgy_gbl_drawer.js
```
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_drawer/ucgy_gbl_drawer.info]: this js, if not needed in all pages should be loaded just in the template

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss (hunk 1 above)
Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss]: It is - I'm mostly through converting and testing the current SCSS file to this one. I ran out of time so just stashed the code.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_drawer/ucgy_gbl_drawer.info (hunk 2 above)
Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_drawer/ucgy_gbl_drawer.info]: This is the way that I've seen it done for most of our existing bricks. If you can point me at an example where we load only when the brick is visible on the page that would be great.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss (hunk 1 above)
Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss]: Actually, I'll just pull it out and stash it on my machine. Get rid of the confusion. ;)

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_drawer/ucgy_gbl_drawer.info (hunk 2 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_drawer/ucgy_gbl_drawer.info]: in the content_ownership_bulk.module: drupal_add_js($path . '/js/dropdown.js');
or in the views-view--ucgy-all-programs-display--block.tpl.php:
drupal_add_js(drupal_get_path('module', 'ucgy_all_program_display').'/js/views-dropdown-style-bootstrap.js');

Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_drawer/ucgy_gbl_drawer.info]: Thanks Mike. This is deeper into the back end than I normally go so happy to learn of this technique as it keeps the page loads and potential conflicts to a minimum!

**Comment by TDRoden** [comment]
I'd like to move the drupal_add_js()/drupal_add_css() call from the .tpl to the .module file as discussed, but discovered it is still loading the assets globally. Should I leave it as is?

That said, I did find a minor text-alignment glitch that I may as well fix now...

**Comment by dpfitzsi** [comment]
Hi Terry. I just merged 962 and now this appears to be in conflict. Can you resolve?

**Comment by TDRoden** [comment]
That should have it cleaned up Dan. Thanks!

**Pull request merged**